SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/OutgoingMessages.cpp

```diff
@@ -11,28 +11,28 @@ void FEntityQueryRequest::TraverseConstraint(Worker_Constraint* Constraint)
 	{
 	case WORKER_CONSTRAINT_TYPE_AND:
 	{
-		TUniquePtr<Worker_Constraint[]> NewConstraints = MakeUnique<Worker_Constraint[]>(Constraint->and_constraint.constraint_count);
+		TUniquePtr<Worker_Constraint[]> NewConstraints = MakeUnique<Worker_Constraint[]>(Constraint->constraint.and_constraint.constraint_count);
 
-		for (unsigned int i = 0; i < Constraint->and_constraint.constraint_count; i++)
+		for (unsigned int i = 0; i < Constraint->constraint.and_constraint.constraint_count; i++)
```

Comment:
not part of this pr but we shouldn't be using `unsigned int` here. If we want to match the types we should use the fixed width ones.

Reply:
Actually this isn't worth fixing as this file is deleted in the worker wrapper stuff.